fb-to-redshift/fb.py

```diff
@@ -75,7 +75,7 @@ def get_posts_and_interactions(interval=False):
                 '%s/posts?fields=message,created_time,id,'
                 'likes.limit(0).summary(total_count),'
                 'comments.limit(0).summary(total_count),'
-                'shares,'
+                'shares,type,permalink_url,'
```

Comment:
not important to this commit, but I might take the opportunity to make this something like:
```
'%(fb_page)s/posts?fields=%(fields)s&limit=%(limit)s...' % {'now': now, 'fb_page': fb_page_id, ...
'fields': ','.join(['message', 'created_time', 'id', 'likes.limit(0).summary(total_count)', .....])}
```
just so we don't worry about comma-merges

Reply:
good idea